test/unittest/tests/wrappers/mock_connection.cpp

```diff
@@ -5,6 +5,7 @@
  *
  * See the file LICENSE for redistribution information.
  */
+#include <filesystem>
```

Comment:
I don't think you need this include? At the very least I don't see any std::filesystem usages here.